src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-FSharp/Program.fs

```diff
@@ -1,27 +1,52 @@
 namespace Company.WebApplication1
-
+#nowarn "20"
```

Comment:
What's this?

Reply:
I think I can guess, it's ignoring return values?

Reply:
> I think I can guess, it's ignoring return values?

Yeah, we discussed it a bit inside the team, and think that having a `#nowarn` is a bit cleaner, than having to ignore all the calls which return something. Unfortunately, we don't (yet) have anything like C#'s 
 scoped `#pragma warning`.